### ISSUE TITLE
Does anyone else's menyoo crash when switching mp clothes?

### ISSUE THREAD

**Issue by roscoehussein** [MAFINS/MenyooSP#364]
Must be dlc related...

**Comment by ColbyDash** [comment]
Yep, mine crashes too on certain categories I actually have on video as a matter of a fact for when I did a tutorial regarding how to make mp characters using Menyoo. In fact, I also have a list of the categories that the game crashes in:
- Legs ► 144
- Shoes ► 122
- Emblem ► 102

Note the shoes and emblem numbers are a rough estimate I roughly remembered crashing around at the time. The game also crashes when you go too far when scrolling through Michael's face textures too.

**Comment by mistahwolf** [comment]
It's an issue with MP Clothes. HeySlickThatsMe hasn't tried to figure out what the problem is as he's moved on from the mod (eyeroll.) Anyway, the workaround is to spawn in an MP Ped (M/F), do the initial dressing and then 'soul steal' making that your playable ped. Don't delete the new ped from the database (there will be a 'freemode male/female' and 'self') and you should be able to change outfits 3-5x without crashing the game. Until 'Slick' or somebody gets around to a fix, it is what it is....

**Comment by mistahwolf** [comment]
Yeah, since I put in the new dlc folders, the game crashes when trying to scroll through clothing for mp characters. Looking at the scripthook and menyoo logs, those aren't the culprits. Instead it's our old friend 'Kernel error' with GTAV.exe. So it's something in the game that's causing the crash.
